src/dotnet/commands/dotnet-run/LocalizableStrings.cs

```diff
@@ -16,26 +16,14 @@ internal class LocalizableStrings
 
         public const string RunCommandException = "The build failed. Please fix the build errors and run again.";
 
-        public const string RunCommandMSBuildExtensionsPath = "MSBuildExtensionsPath";
-
-        public const string RunCommandConfiguration = "Configuration";
-
-        public const string RunCommandTargetFramework = "TargetFramework";
-
-        public const string RunCommandProjectInstance = "RunCommand";
-
-        public const string RunCommandOutputType = "OutputType";
+        public const string RunCommandExceptionUnableToRun = "Unable to run your project\nPlease ensure you have a runnable project type and ensure '{0}' supports this project.\nThe current {1} is '{2}'";
```

Comment:
Missing a period after project. According to Microsoft's style guide, you should avoid using please unless you're asking something to the user that is inconvenient or the app is to blame for this. 

Suggested change:
"Unable to run your project.\n Ensure you have a runnable project type and that '{0}' supports this project.\nThe current {1} is '{2}'."